docs/reference/glossary.md

```diff
@@ -483,203 +404,216 @@ Changes to the world are made by [workers](#worker). Each worker has a view onto
 
-### Unity Project
+## Unity Project
 
-> Not to be confused with [SpatialOS Project](#spatialos-project) or [project name](#project-name).
+<span style="display: block;margin-top: -20px;font-size: 10pt;"><i>Not to be confused with [SpatialOS Project](#spatialos-project) or [project name](#project-name).</i></span>
 
-A Unity project is the source code and assets of a SpatialOS game's Unity [workers](#worker). We often use this to refer to the directory that contains Unity code and assets, `<path to the GDK repo>/workers/unity` in this case.
+A Unity project is the source code and assets of a SpatialOS game's Unity [workers](#worker).
 
-### Update Loop
+## Update Loop
 
 The Unity ECS updates all systems on the main thread. The order in which they are updated is based on constraints that you can add to your systems.
 
-> Related:
+> **Related:**
 >
-> * [System update order (Unity documentation)](https://docs.unity3d.com/Packages/com.unity.entities@0.0/manual/system_update_order.html)
+> * [System update order](https://docs.unity3d.com/Packages/com.unity.entities@0.0/manual/system_update_order.html)
 > * [System update order in the GDK]({{urlRoot}}/reference/workflows/ecs/concepts/system-update-order)
 
-### Worker
+## Worker
 
-The [SpatialOS Runtime](#spatialos-runtime) manages the [SpatialOS world](#spatialos-world): it keeps track of all the [SpatialOS entities](#spatialos-entity) and their
-[SpatialOS components](#spatialos-component). But on its own, it doesn’t make any changes to the world.
+Workers are server-side and client-side software. They perform the computation associated with a [SpatialOS world](#spatialos-world): they are responsible for iterating on the game simulation and updating the definitive state of the game.
 
-Workers are programs that connect to a SpatialOS world. They perform the computation associated with a world:
-they can read what’s happening, watch for changes, and make changes of their own.
+### Worker types and worker instances
 
-There are two types of workers, [server-workers](#server-worker) and [client-workers](#client-worker).
+When you develop your game, you set up worker types. These are like molds for the worker instances which actually do the computation.
 
-In order to achieve huge scale, SpatialOS divides up the SpatialOS entities in the world between workers, balancing the work so none of them are overloaded. For each SpatialOS entity in the world, it decides which worker should have [write access](#write-access) to each SpatialOS component on the SpatialOS entity. To prevent multiple workers writing to a component at the same time, only one worker at a time can have write access to a SpatialOS component.
+Worker instances are the programs that compute and connect a SpatialOS world. In general, you use one or more server-worker instances to compute the world, and each player’s client software uses a client-worker instance to interact with the world.
```

Comment:
```suggestion
Worker instances are the programs that compute and connect a SpatialOS world. In general, you use one or more server-worker instances to simulate the world, and each player’s client software uses a client-worker instance to interact with the world.
```

I personally prefer simulate to compute, but it's not a strong opinion

Reply:
This was copy-pasta'ed from the other glossary, so I'd take that one up with @ElleEss or @anne-edwards

Reply:
in that case just leave it as it is